doc/programming_guide/ffd/speech_recognition_cyberon.rst

```diff
@@ -22,7 +22,8 @@ The Cyberon DSpotter™ speech recognition engine runs proprietary models to ide
 
 One model for US English is provided. For any technical questions or additional models please contact Cyberon.
 
-To replace the Cyberon engine with a different engine, refer to the ASR documentation on :ref:`sln_voice_asr_programming_guide`
+.. TODO: Check if the line below can be removed or re-added
```

Comment:
Will this ToDo be completed as part of this PR?  If not, let's capture it in a Jira issue somewhere.